src/AutoRest.CSharp/Common/Generation/Types/TypeFactory.cs

```diff
@@ -465,6 +525,13 @@ private string GetFullMetadataName(ISymbol namedTypeSymbol)
 
         private void GetFullMetadataName(StringBuilder builder, ISymbol symbol)
```

Comment:
I suggest we rename this to `BuildFullMetadataName` to avoid confusion

Reply:
makes sense